java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -172,17 +174,28 @@ private void handleBlockExit(ProgramPoint programPosition) {
         case IFLT:
         case IFGE:
         case IFGT:
-        case IFLE:
-
+        case IFLE: {
+          SymbolicValue svZero = new SymbolicValue();
+          ProgramState.Pop pop = programState.unstackValue(1);
+          List<ProgramState.SymbolicValueSymbol> symbolicValueSymbols = new ArrayList<>(pop.valuesAndSymbols);
+          symbolicValueSymbols.add(new ProgramState.SymbolicValueSymbol(svZero, null));
+          List<ProgramState> programStates = svZero.setConstraint(pop.state, DivisionByZeroCheck.ZeroConstraint.ZERO).stream()
+            .flatMap(s -> svZero.setConstraint(s, BooleanConstraint.FALSE).stream()).collect(Collectors.toList());
+          Preconditions.checkState(programStates.size() == 1);
+          programState = programStates.get(0).stackValue(constraintManager.createBinarySymbolicValue(terminator, symbolicValueSymbols));
+        }
+        break;
         case IF_ICMPEQ:
         case IF_ICMPNE:
         case IF_ICMPLT:
         case IF_ICMPGE:
         case IF_ICMPGT:
         case IF_ICMPLE:
         case IF_ACMPEQ:
-        case IF_ACMPNE:
-
+        case IF_ACMPNE: {
+          ProgramState.Pop pop = programState.unstackValue(2);
+          programState = pop.state.stackValue(constraintManager.createBinarySymbolicValue(terminator, pop.valuesAndSymbols));
+        }
           break;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)